src/modules/chatExtras/main.ts

```diff
@@ -11,11 +11,29 @@ export default (async ({ LSSM, getSetting }) => {
         }
     });
 
-    getSetting('cloneHistoryBtnToHeader').then(clone => {
+    if(location.pathname === '/') getSetting('cloneHistoryBtnToHeader').then(clone => {
         if (clone) {
             import(
                 /* webpackChunkName: "modules/chatExtras/cloneHistoryBtnToHeader" */ './assets/cloneHistoryBtnToHeader'
             ).then(({ default: cloner }) => cloner());
         }
     });
+
+    getSetting('lightDesignChatHistory').then((lightChatDesignActive)  => {
+      if(lightChatDesignActive){
+        import(
+            /* webpackChunkName: "modules/chatExtras/lightDesignChatHistory" */ './assets/lightDesignChatHistory'
+        ).then(({default: addLightChatDesign}) => {
+          if(location.pathname.includes('alliance_chats')) {
+            addLightChatDesign(false, '');
+          }
+          LSSM.$store.dispatch('event/addListener', {
+            name: 'redesign-finished-loading',
```

Comment:
What happens if the redesign isnt active?

Reply:
it will add the design all the time (if the page is the chat history) before the redesign loads and, if the redesign is active, after the redesign loads; see ll. 32-34